tracer/tools/Samples.Transport.UnixDomainSocket/Samples.Transport.UnixDomainSocket.csproj

```diff
@@ -29,7 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.14.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.14.2" />
```

Comment:
is this change intended?

Reply:
good spot, rolled back in 0b9ad25e7b9c01930daf80f0c6c3ced029d69c5c